### PR TITLE
Re-activate and Update Statistics.gov.uk.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/statistics.gov.uk.xml [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
-			<test url="http://www.statistics.gov.uk/index.php" />
-
-
-	<securecookie host="^\w" name=".+" />
-
-
-	<!--	Redirect drops forward slash and args:
-							-->
-	<rule from="^http://www\.statistics\.gov\.uk/.*"
-		to="https://www.gov.uk/government/statistics/announcements" />
```
Comment (J0WI) [inline src/chrome/content/rules/statistics.gov.uk.xml]: Shouldn't we keep that one?

File: src/chrome/content/rules/Statistics.gov.uk.xml [hunk 2]
```diff
@@ -29,8 +29,14 @@
 		- www2.secure-transfer.statistics.gov.uk
 -->
 <ruleset name="Statistics.gov.uk (partial)">
+	<target host="www.statistics.gov.uk" />
+		<test url="http://www.statistics.gov.uk/about/national_statistics" />
 	<target host="neighbourhood.statistics.gov.uk" />
 	<target host="www.neighbourhood.statistics.gov.uk" />
+	
+	<rule from="^http://www\.statistics\.gov\.uk/.*"
+		  	to="https://www.gov.uk/government/statistics/announcements" />
```
Comment (J0WI) [inline src/chrome/content/rules/Statistics.gov.uk.xml]: That behavior seems wrong: `http://www.statistics.gov.uk/invalid`

Comment (cschanaj) [reply src/chrome/content/rules/Statistics.gov.uk.xml]: Do you mean that I should also include the exclusion in the old ruleset?

Comment (J0WI) [reply src/chrome/content/rules/Statistics.gov.uk.xml]: Yes, or you change the rule to only match specific urls.
And please remove is the `.*` in the expression.

**Comment by J0WI** [comment]
Please check the test coverage exposed by Travis.

**Comment by cschanaj** [comment]
@J0WI Travis passing now. thanks!

**Pull request merged**